java/ql/src/Security/CWE/CWE-798/HardcodedCredentialsApiCall.qhelp

```diff
@@ -32,13 +32,43 @@
     Instead, the user name and password could be supplied through environment variables,
     which can be set externally without hard-coding credentials in the source code.
   </p>
+
+  <p>
+    The following code example connects to AWS using a hard-coded access key ID and secret key:
+  </p>
+
+  <sample src="HardcodedAWSCredentials.java"/>
+
+  <p>
+    Instead, the access key ID and secret key could be supplied through environment variables,
+    which can be set externally without hard-coding credentials in the source code.
+  </p>
+
+  <p>
+    The following code example connects to Azure using a hard-coded user name and password or client secret:
+  </p>
+
+  <sample src="HardcodedAzureCredentials.java"/>
+
+  <p>
+    Instead, the username and password or client secret could be supplied through environment variables,
+    which can be set externally without hard-coding credentials in the source code.
+  </p>
```

Comment:
No need to change the qhelp, the query is already generically trying to find hardcoded creds; there's no need for it to exhaustively show each one.

Reply:
I've added a change note and reset the qhelp file.